### PR TITLE
fix(core): unique constraint to vos shortName

### PR DESCRIPTION
**Pull request by sarkapalkovicova**
Dropped unique constraint to name and added a constraint to shortName instead.

### CODE REVIEW

File: perun-core/src/main/resources/postgresChangelog.txt [hunk 1]
```diff
@@ -6,6 +6,10 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.2.11
+ALTER TABLE vos DROP constraint vo_u unique (name);
```
Comment (HejdaJakub) [inline perun-core/src/main/resources/postgresChangelog.txt:10]: I think that if you drop the constraint you should just define a name without the rest of the definition part.
```suggestion
ALTER TABLE vos DROP constraint vo_u;
```

File: perun-base/src/test/resources/test-schema.sql [hunk 2]
```diff
@@ -1881,7 +1881,7 @@ create index idx_fk_attr_critops ON attribute_critical_actions(attr_id);
 create index app_state_idx ON application (state);
 
 -- set initial Perun DB version
-insert into configurations values ('DATABASE VERSION','3.2.10');
+insert into configurations values ('DATABASE VERSION','3.2.11');
```
Comment (HejdaJakub) [inline perun-base/src/test/resources/test-schema.sql:1884]: Update also comment on the first line of this file, please.

File: perun-db/postgres.sql [hunk 3]
```diff
@@ -1988,7 +1988,7 @@ grant all on attribute_critical_actions to perun;
 grant all on app_notifications_sent to perun;
 
 -- set initial Perun DB version
-insert into configurations values ('DATABASE VERSION','3.2.10');
+insert into configurations values ('DATABASE VERSION','3.2.11');
```
Comment (HejdaJakub) [inline perun-db/postgres.sql:1991]: The same. Update also comment on the first line of this file, please.

**Review by HejdaJakub — CHANGES_REQUESTED** [review]

**Review by HejdaJakub — APPROVED** [review]

**Review by xflord — COMMENTED** [review]
Mark the commit as breaking change, as we're updating DB (you can also add this to release notes for deployment)

**Review by xflord — APPROVED** [review]

**Pull request merged**